QuickViewer/src/mainwindow.ui

```diff
@@ -687,46 +687,46 @@
      <normaloff>:/icons/volume_prev_icon</normaloff>:/icons/volume_prev_icon</iconset>
    </property>
    <property name="text">
-    <string extracomment="Switch to the previous Volume in the same folder as Volume currently being displayed Button">P&amp;rev Volume</string>
+    <string extracomment="Switch to the previous Volume in the same folder as Volume currently being displayed Button">P&amp;revious volume</string>
    </property>
   </action>
   <action name="actionNextOnePage">
    <property name="text">
-    <string extracomment="Because there are times when the page is missing due to bookbinding convenience, digital books will fit the left and right pages with this operation">Next &amp;one page only</string>
+    <string extracomment="Because there are times when the page is missing due to bookbinding convenience, digital books will fit the left and right pages with this operation">Next &amp;single page</string>
```

Comment:
I didn't notice any changes clicking this. Does this works in Windows?

Reply:
If you have two pages of manga or other scrapbooks open in QuickViewer, QuickViewer will switch pages two pages at a time. This feature forces you to switch only one page. This makes it possible to correct the phenomenon that the left and right pages are reversed.